flocker/dockerplugin/_api.py

```diff
@@ -28,7 +28,7 @@
 from ..control._config import dataset_id_from_name
 from ..apiclient import DatasetAlreadyExists
 from ..node.agents.blockdevice import PROFILE_METADATA_KEY
-
+from .testtools import parse_num
```

Comment:
If you are going to be using this in non-test code, then the implementation should be moved out of `testtools`.